### PR TITLE
Name normalization etc

### LINKED ISSUE #60: Change the doc section file names to work areas
Right now the files (and URLs generated as a result) are mostly subject gerunds (e.g., "https://datarefuge.github.io/workflow/harvesting/") but there are some outliers (advance work, coordinator work, research, metadata, index [but this one has to stay for mkdocs])

advance-work.md
bagging.md
checking.md
coordinator-work.md
harvesting.md
index.md 
metadata.md
research.md
seednsort.md

In the overview sections are referred to by role (i.e., Harvesters, Baggers)

Can we pick one and be consistent with how the pages are referenced? My vote would be to have the page urls/headings describe the area of work ("researching"), not the role ("researcher")

### PR DESCRIPTION
**Pull request by khdelphine**
* Normalized file names and use of gerunds in headings -- Fixes #60
* Added mention of the DataRefuge slack-in

### CODE REVIEW

File: docs/describing.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ Describers creates a descriptive record in the DataRefuge CKAN repository for ea
     - Note that an email address is required to apply.
     - Note also that you should be willing to have your real name be associated with the datasets, to follow archival best practices (see [guidelines on archival best practices for Data Refuge](http://www.ppehlab.org/blogposts/2017/2/1/data-refuge-rests-on-a-clear-chain-of-custody) for more information).
 - The organizers of the event (in-person or remote) will send you an invite to the [Archivers app](http://www.archivers.space/), which helps us coordinate all the data archiving work we do.
-  	- Click the invite link, and choose a user name and a password.
-- Make sure you have an account on the DataRefuge slack (or other slack team recommended by your event organizers) This is where people share expertise and answer each other's questions.
-	- Ask your event organizer to send you an invite
+    - Click the invite link, and choose a user name and a password.
+- Create an account on the DataRefuge slack using this [slack-in](https://rauchg-slackin-qonsfhhvxs.now.sh/)) or use the slack team recommended by your event organizers. This is where people share expertise and answer each other's questions.
```
Comment (dcwalk) [inline docs/describing.md]: extra parenthesis ")"

File: docs/harvesting.md [hunk 2]
```diff
@@ -25,8 +25,7 @@ Harvesters take the "uncrawlable" data and try to figure out how to actually cap
 - The organizers of the event (in-person or remote) will tell you how to volunteer for the Harvester role, either through slack or a form.
     - As a result, they will send you an invite to the [Archivers app](http://www.archivers.space/), which helps us coordinate all the data archiving work we do.
     - Click the invite link, and choose a user name and a password.
-- Make sure you have an account on the DataRefuge slack (or other slack team recommended by your event organizers) This is where people share expertise and answer each other's questions.
-- Ask your event organizer to send you an invite
+- Create an account on the DataRefuge slack using this [slack-in](https://rauchg-slackin-qonsfhhvxs.now.sh/)) or use the slack team recommended by your event organizers. This is where people share expertise and answer each other's questions.
```
Comment (dcwalk) [inline docs/harvesting.md]: extra parenthesis ")"

File: docs/index.md [hunk 3]
```diff
@@ -21,34 +21,33 @@ We are so glad that you are participating in this project!
 
 ## Plan Overview
 
-### 1. [Seeders](seeding.md)
+### 1. [Seeding](seeding.md)
 
 Seeders canvass the resources of a given government agency, identifying important URLs. They identify whether those URLs can be crawled by the Internet Archive's webcrawler. If the URLs are crawlable, the Seeders nominate them to the End-of-Term (EOT) project, otherwise they add them to the Uncrawlable spreadsheet using the project's Chrome Extension.
 
-### 2. [Researchers](research.md)
+### 2. [Researching](research.md)
```
Comment (dcwalk) [inline docs/index.md]: File should be renamed `researching.md`

File: docs/index.md [hunk 4]
```diff
@@ -21,34 +21,33 @@ We are so glad that you are participating in this project!
...
 
-### 3. [Harvesters](harvesting.md)
+### 3. [Harvesting](harvesting.md)
 
 Harvesters take the "uncrawlable" data and try to figure out how to actully capture it based on the recommendations of the Researchers. This is a complex task which can require substantial technical expertise, and which requires different techniques for different tasks. Harvesters should see the included [Harvesting Toolkit](harvesting.md) for more details and tools.
 
-### 4. Checkers
+### 4. Checking
 
 <!--### 4. [Checkers](checking.md)-->
```
Comment (dcwalk) [inline docs/index.md]: Can we delete this comment? No longer useful

File: docs/index.md [hunk 5]
```diff
@@ -21,34 +21,33 @@ We are so glad that you are participating in this project!
...
 
 Checkers inspect a harvested dataset and make sure that it is complete. The main question the checkers need to answer is "will the bag make sense to a scientist"? Checkers need to have an in-depth understanding of harvesting goals and potential content variations for datasets.
 
-### 5. [Baggers](bagging.md)
+### 5. [Bagging](bagging.md)
 
 Baggers perform some quality assurance on the dataset to make sure the content is correct and corresponds to the original URL. Then they package the data into a bagit file (or "bag"), which includes basic technical metadata and upload it to final DataRefuge destination.
 
-### 6. [Describers](metadata.md)
+### 6. [Describing](metadata.md)
```
Comment (dcwalk) [inline docs/index.md]: Link should be `describing.md`

File: docs/researching.md [hunk 6]
```diff
@@ -12,8 +12,7 @@ Researchers review "uncrawlables" identified during [Seeding](seeding.md), confi
 - Event organizers (in-person or remote) will tell you how to volunteer for the Researcher role, either through slack or a form.
     - As a result, they will send you an invite to the [Archivers app](http://www.archivers.space/), which helps us coordinate all the data archiving work we do.
     - Click the invite link, and choose a user name and a password.
-- Make sure you have an account on the DataRefuge slack (or other slack team recommended by your event organizers) This is where people share expertise and answer each other's questions.
-    - Ask your event organizer to send you an invite
+- Create an account on the DataRefuge slack using this [slack-in](https://rauchg-slackin-qonsfhhvxs.now.sh/)) or use the slack team recommended by your event organizers. This is where people share expertise and answer each other's questions.
```
Comment (dcwalk) [inline docs/researching.md]: extra parenthesis ")"

**Review by dcwalk — COMMENTED** [review]
It looks like the file names weren't changed

**Comment by khdelphine** [comment]
Sorry, I had not do a complete job on this.  Hopefully it should be good now.

**Review by dcwalk — APPROVED** [review]

**Pull request merged**